flaml/tune/tune.py

```diff
@@ -735,9 +737,14 @@ def easy_objective(config):
                         )
                         results = None
                         with PySparkOvertimeMonitor(time_start, time_budget_s, force_cancel, parallel=parallel):
-                            results = parallel(
-                                delayed(evaluation_function)(trial_to_run.config) for trial_to_run in trials_to_run
-                            )
+                            try:

```

Comment:
Hi Li, context manager could handle exception, you can catch the exception in [`__exit__`](https://github.com/microsoft/FLAML/blob/main/flaml/tune/spark/utils.py#L286) of  `PySparkOvertimeMonitor`.

Reply:
Cool, thanks for reminding! Updated.